### DISCUSSION TITLE
Should this work when using another network?

### DISCUSSION THREAD

**Discussion by taimansulimani** [unknownskl/xbox-xcloud-client#621] [Q&A]
this is great, xbox remote play works fine on my mac using this client when I'm on the same wifi network.
but it doesn't work for me if I'm connected to another wifi network or 4g. it sees my xbox, it connects, but the video stream is not loaded. xbox is connected to my network via ethernet.

thing is remote play works fine over 4g with the official android app

has anyone experienced the same?

**Comment by rackerby** [comment]
I have experienced the same.  Remote Play in the official Xbox app for iOS stopped working via cellular-data with iOS 16, so there may be something similar going on with macOS.

**Reply by rackerby** [reply]
I take it back... Remote Play does appear to work on macOS tethered to cellular-data if I use version 1.2.0 of xbox-xcloud-client.  It doesn't work with Greenlight.

**Reply by taimansulimani** [reply]
oh really? I just tried xbox-cloud-client 1.2.0 and I can connect but I see a black screen, just like with Greenlight. 
Is your xbox network using IPv6?

**Comment by rackerby** [comment]
Yes...  IPv6 all around.

**Reply by rackerby** [reply]
Ah.  This only works when connecting to my One X.  It does not work when connecting to my Series X.

**Reply by rackerby** [reply]
Possibly related:
https://answers.microsoft.com/en-us/xbox/forum/all/remote-play-on-ios-devices-known-issue/042131a3-d7c0-4df9-9068-5864ea3e89f2

**Reply by taimansulimani** [reply]
I was told it doesn't work because of the lack of IPv6, but then you're saying that your Series X is using IPv6 and still doesn't work so I don't really know what's going on anymore!

https://www.reddit.com/r/XboxSeriesX/comments/wfzrnt/you_can_now_remote_play_on_macs_with_new/izwpjmu?utm_medium=android_app&utm_source=share&context=3
